### PR TITLE
enhance trackle message payload doc

### PR DESCRIPTION
**Pull request by leroxyl**

### CODE REVIEW

File: README_PAYLOAD.md [hunk 1]
```diff
@@ -61,21 +61,26 @@ todo
...
-    "i": 60000,
-    "il": 1800000
-   }
+
+trackle message payload:
+```
+  [ // <msgpack.array[5]
+    "v1.0.2-PROD-20180326103205 (v5.6.6)", // <msgpack.raw> Version String
+    5, // <msgpack.uint32> wakeup cycles
+    3, // <msgpack.unit32> error state
```
Comment (lalottacontinua) [inline README_PAYLOAD.md]: is there a documentation/link to the potential error states that exist?

File: README_PAYLOAD.md [hunk 2]
```diff
@@ -61,21 +61,26 @@ todo
...
-    "max": 4200,
-    "i": 60000,
-    "il": 1800000
-   }
+
+trackle message payload:
+```
+  [ // <msgpack.array[5]
+    "v1.0.2-PROD-20180326103205 (v5.6.6)", // <msgpack.raw> Version String
+    5, // <msgpack.uint32> wakeup cycles
```
Comment (lalottacontinua) [inline README_PAYLOAD.md]: is there some description about what exactly a wakeup count will trigger?

**Review by lalottacontinua — APPROVED** [review]
LGTM! Just added some questions, in case you're at the source of information.. :)

File: README_PAYLOAD.md (hunk 1 above)
Comment (leroxyl) [reply README_PAYLOAD.md]: apparently there is not. Looking at the source code, the status consists of 4 bytes, where 6 different flags are OR-combined. 
0x03 means that the device shelf test has been executed and keys have been generated, i.e. 0x03 means the device is ready

File: README_PAYLOAD.md (hunk 2 above)
Comment (leroxyl) [reply README_PAYLOAD.md]: the wakeup cycle counter is triggered once every measuring interval. the interval duration is part of the configuration map. it is the "i" value in milliseconds

**Pull request merged**